examples/intRVFL.py

```diff
@@ -0,0 +1,539 @@
+        "HorseColic",
+        "IlpdIndianLiver",
+        "ImageSegmentation",
+        "Ionosphere",
+        "Iris",
+        "LedDisplay",
+        "Lenses",
+        "Letter",
+        "Libras",
+        "LowResSpect",
+        "LungCancer",
+        "Lymphography",
+        "Magic",
+        "Mammographic",
+        "Miniboone",
+        "MolecBiolPromoter",
+        "MolecBiolSplice",
+        "Monks1",
+        "Monks2",
+        "Monks3",
+        "Mushroom",
+        "Musk1",
+        "Musk2",
+        "Nursery",
+        "OocytesMerlucciusNucleus4d",
+        "OocytesMerlucciusStates2f",
+        "OocytesTrisopterusNucleus2f",
+        "OocytesTrisopterusStates5b",
+        "Optical",
+        "Ozone",
+        "PageBlocks",
+        "Parkinsons",
+        "Pendigits",
+        "Pima",
+        "PittsburgBridgesMaterial",
+        "PittsburgBridgesRelL",
+        "PittsburgBridgesSpan",
+        "PittsburgBridgesTOrD",
+        "PittsburgBridgesType",
+        "Planning",
+        "PlantMargin",
+        "PlantShape",
+        "PlantTexture",
+        "PostOperative",
+        "PrimaryTumor",
+        "Ringnorm",
+        "Seeds",
+        "Semeion",
+        "Soybean",
+        "Spambase",
+        "Spect",
+        "Spectf",
+        "StatlogAustralianCredit",
+        "StatlogGermanCredit",
+        "StatlogHeart",
+        "StatlogImage",
+        "StatlogLandsat",
+        "StatlogShuttle",
+        "StatlogVehicle",
+        "SteelPlates",
+        "SyntheticControl",
+        "Teaching",
+        "Thyroid",
+        "TicTacToe",
+        "Titanic",
+        "Trains",
+        "Twonorm",
+        "VertebralColumn2Clases",
+        "VertebralColumn3Clases",
+        "WallFollowing",
+        "Waveform",
+        "WaveformNoise",
+        "Wine",
+        "WineQualityRed",
+        "WineQualityWhite",
+        "Yeast",
+        "Zoo",
+    ]
+
+    def __init__(
+        self,
+        model_class: nn.Module,
+        simul: int = 1,
+        batch_size: int = 1,  # DK: The current code will not work for other values. Because "encoding_density_clipped()" assumes only one sample. Do we want to make it more generic?
+        device: torch.device = torch.device("cpu"),
+    ):
+        self.model_class = model_class
+        self.simul = simul
+        self.batch_size = batch_size
+        self.device = device
+
+        # Collects average accuracy for each simulation of each dataset
+        self.accuracy_collection = torch.zeros(
+            len(self.dataset_collection),
+            self.simul,
+            device=self.device,
+        )
+
+    ## Function compute the accuracy for a given model and data
+    def compute_accuracy(self, model, train_ds, test_ds):
+        # Make data loaders
+        train_ld = torch.utils.data.DataLoader(
+            train_ds, batch_size=self.batch_size, shuffle=True
+        )
+        test_ld = torch.utils.data.DataLoader(
+            test_ds, batch_size=self.batch_size, shuffle=True
```

Comment:
There is no need to shuffle the samples in the test set

Reply:
Revised